starknetdata/gateway/gateway.go

```diff
@@ -307,7 +307,7 @@ func (g *Gateway) StateUpdate(ctx context.Context, blockNumber uint64) (*core.St
 
 func AdaptStateUpdate(response *clients.StateUpdate) (*core.StateUpdate, error) {
 	stateDiff := new(core.StateDiff)
-	stateDiff.DeclaredContracts = response.StateDiff.DeclaredContracts
+	stateDiff.DeclaredClasses = response.StateDiff.DeclaredContracts
```

Comment:
I think the lingo here is `DeclaredContracts = Classes`. So, we either go with just `Classes` or `DeclaredContracts`

Reply:
In this case, I think it is preferable to emphasize that these classes are declared in this state diff. Since classes are declared and contracts are deployed, `DeclaredClasses` seems to be a fitting name.